tests/deployment_utils.py

```diff
@@ -223,15 +223,20 @@ def _create_temp_config(self, config: str):
     def run(self, config: str, force_installed: bool = False, vcs_type: str = "none",
             additional_parameters="", environment=None, expected_to_fail=False, workdir=None):
 
-        if utils.is_pycharm():
-            cmd = "{0}/universum/__main__.py".format(self.working_dir)
+        # workdir is ignored unless force_installed;
+        # if installed to make sure local sources are not used by default the directory is changed
+        if force_installed:
+            if not workdir:
+                workdir = os.path.join(self.working_dir, "another_directory")
```

Comment:
I don't understand where this"another_directory" is coming from and what is the goal.
We already have quite dirty working directory after running tests.

Reply:
The thing is, `python -m module_name` is a little bit dirty itself. If module is installed in system, this command will work from any place on computer; if module _is not installed,_ it will work only from the directory where module's sources are located.
So if we want to make sure Universum was installed correctly, we _have_ to launch it from anywhere else than workdir.

Reply:
Then please update the comment to make the intention more clear.